chrome-extension/content.js

```diff
@@ -70,40 +65,46 @@ function closeSidebar() {
 function createSidebar(content) {
   sidebar.innerHTML = '';
   sidebar.style.display = 'block';
-  const panelHeading = document.createElement('div');
-  panelHeading.classList.add('message-header');
-  panelHeading.innerText = 'Lobster Ice Cream';
+
+  const panelHeading = customCreateElement('div', ['message-header'], 'Lobster Ice Cream');
   //TODO: Include icon in panelhead, update looks
 
-  const close = document.createElement('a');
-  close.classList.add('delete');
+  const close = customCreateElement('a', ['delete']);
   close.onclick = closeSidebar;
   panelHeading.appendChild(close);
   sidebar.appendChild(panelHeading);
 
   const form = document.createElement('form');
-  const highlightLabel = document.createElement('label');
-  highlightLabel.classList.add('label', 'mt-2');
-  highlightLabel.innerText = 'Highlighted Text';
+
+  const highlightLabel = customCreateElement('label', ['label', 'mt-2'], 'Highlighted Text');
   form.appendChild(highlightLabel);
 
-  const highlightTextarea = document.createElement('textarea');
-  highlightTextarea.classList.add('textarea');
+  const highlightTextarea = customCreateElement('textarea', ['textarea']);
   form.appendChild(highlightTextarea);
   highlightTextarea.value = content;
 
-  const noteTextarea = document.createElement('textarea');
-  noteTextarea.classList.add('textarea', 'mt-6');
+  const noteTextarea = customCreateElement('textarea', ['textarea', 'mt-6']);
   noteTextarea.setAttribute('placeholder', 'Note to self');
   form.appendChild(noteTextarea);
 
-  const buttonContainer = document.createElement('div');
-  buttonContainer.classList.add('has-text-centered', 'mt-1');
-  const addButton = document.createElement('button');
-  addButton.classList.add('button', 'is-link');
-  addButton.innerHTML = 'Add Clipping';
+  const buttonContainer = customCreateElement('div', ['has-text-centered', 'mt-1']);
+
+  const addButton = customCreateElement('button', ['button', 'is-link'], 'Add Clipping');
   buttonContainer.appendChild(addButton);
   form.appendChild(buttonContainer);
 
   sidebar.appendChild(form);
 }
+
+/**
+ * Helper to create an HTML Element
+ * @param {String} type
+ * @param {List of Strings} classList
+ * @param {String} innerHTML
+ */
+function customCreateElement(type, classList, innerHTML = '') {
```

Comment:
Feel free to ignore this, but I was wondering if it would be fine to just call this createElement? If you want it to be distinct, it could also be generateElement or fillElement.

Reply:
ah so i chose customCreateElement because usually it's document.createElement, so... customized lol